src/core/Canvas.js

```diff
@@ -11,17 +11,25 @@ export class Canvas extends BaseCanvas {
 
     constructor (config, eventBus, graphicsFactory, elementRegistry) {
         super(config, eventBus, graphicsFactory, elementRegistry);
-        this.createUiGroup();
+        this._uiGroup = this.createUiGroup();
```

Comment:
Why the underscore?

Reply:
Mainly to be consistent with the parent class.